src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  height: 100%;
```

Comment:
The height of 100% might not work as expected if the parent element does not have a specified height. Make sure that the parent element has a defined height.